src/main/java/ui/UI.java

```diff
@@ -350,14 +351,18 @@ private Parent createRootNode() {
     public Set<String> getCurrentlyUsedRepos() {
         Set<String> currentlyUsedRepos = new HashSet<>();
         String defaultRepo = logic.getDefaultRepo();
-        currentlyUsedRepos.add(defaultRepo);
         currentlyUsedRepos.addAll(panels.getRepositoriesReferencedOnAllPanels());
+        if (!currentlyUsedRepos.stream().map(String::toLowerCase)
+                .collect(Collectors.toSet()).contains(defaultRepo.toLowerCase())) {
+            currentlyUsedRepos.add(defaultRepo);
+        }
```

Comment:
Yep, will make normalize in Utility and add comment on it that should be removed once #1078 is done.